pkg/apis/core/v1/appconfiguration.go

```diff
@@ -20,61 +20,61 @@ type Accessory map[string]interface{}
-//		   }
+//	  helloWorld: ac.AppConfiguration {
+//	      # Built-in module
+//	      workload: wl.Service {
+//	          containers: {
+//	              "main": c.Container {
+//	                  image: "ghcr.io/kusion-stack/samples/helloworld:latest"
+//	                  # Configure a HTTP readiness probe
+//	                  readinessProbe: p.Probe {
+//	                      probeHandler: p.Http {
+//	                          url: "http:localhost:80"
+//	                      }
+//	                  }
+//	              }
+//	          }
+//	      }
 //
-//		   # extend accessories module base
+//	  # a collection of accessories that will be attached to the workload
 //	    accessories: {
-//	       # Built-in module, key represents the module source
-//	       "kusionstack/mysql@v0.1" : d.MySQL {
-//	           type: "cloud"
-//	           version: "8.0"
-//	       }
-//	       # Built-in module, key represents the module source
-//	       "kusionstack/prometheus@v0.1" : m.Prometheus {
-//	           path: "/metrics"
-//	       }
-//	       # Customized module, key represents the module source
-//	       "foo/customize": customizedModule {
-//	               ...
-//	       }
-//	   }
+//	        # Built-in module, key represents the module source
+//	        "kusionstack/mysql@v0.1.0" : d.MySQL {
+//	            type: "cloud"
+//	            version: "8.0"
+//	        }
+//	        # Built-in module, key represents the module source
+//	        "kusionstack/prometheus@v0.1.0" : m.Prometheus {
+//	            path: "/metrics"
+//	        }
+//	        # Customized module, key represents the module source
+//	        "foo/customize@v0.1.0": customizedModule {
+//	                ...
+//	        }
+//	    }
 //
-//		   # extend pipeline module base
-//		   pipeline: {
-//		       # Step is a module
-//		       "step" : Step {
-//		           use: "exec"
-//		           args: ["--test-all"]
-//		       }
-//		   }
+//	      # pipeline modules
+//	      pipeline: {
+//	          # Step is a module
+//	          "step" : Step {
+//	              use: "exec"
+//	              args: ["--test-all"]
+//	          }
+//	      }
 //
-//		   # Dependent app list
-//		   dependency: {
-//		       dependentApps: ["init-kusion"]
-//		   }
-//		}
+//	      # dependent app list
+//	      dependency: {
+//	          dependentApps: ["init-kusion"]
+//	      }
+//	}
 type AppConfiguration struct {
 	// Name of the target App.
 	Name string `json:"name,omitempty" yaml:"name,omitempty"`
 	// Workload defines how to run your application code.
 	Workload *workload.Workload `json:"workload" yaml:"workload"`
 	// Accessories defines a collection of accessories that will be attached to the workload.
-	// The key in this map represents the module source. e.g. kusionstack/mysql@v0.1
-	Accessories map[string]*Accessory `json:"accessories,omitempty" yaml:"accessories,omitempty"`
+	// The key in this map represents the module source. e.g. kusionstack/mysql@v0.1.0.0
```

Comment:
Is the example here not in line with the format of semantic versioning (`v0.1.0.0` rather than `v0.1.0` seems to be with an extra part)?